### ISSUE TITLE
Training config

### ISSUE THREAD

**Issue by KeplerrC** [Y-debug-sys/Diffusion-TS#5]
can you provide the training config for the solar dataset in the paper? I see that the visualizations in the paper are excellent!

**Comment by Y-debug-sys** [comment]
Hi, we repeated the experiment of SSSD repository (reusing code at https://github.com/AI4HealthUOL/SSSD/blob/main/docs/instructions/Solar/solar.ipynb), with 65 samples of length 192 as the training set, and the other 8 samples as the test set. The modified config is as follows
> n_layer_enc=4, n_layer_dec=4, d_model=96, timesteps=500, sampling_timesteps=500, train_num_steps=1500, patience=300, warmup=100, sampling_steps=200 and batch_size=32 (drop_last=True in dataloader).

**Comment by KeplerrC** [comment]
Hi author, can you please provide me with the training code or script, I found that I can't reproduce the results of this experiment
